config/connection.js

```diff
@@ -0,0 +1,37 @@
+const mysql = require("mysql");
+
+class Database {
+  constructor( config ) {
+      this.connection = mysql.createConnection( config );
+  }
+  query( sql, args ) {
+      return new Promise( ( resolve, reject ) => {
+          this.connection.query( sql, args, ( err, rows ) => {
+              if ( err )
+                  return reject( err );
+              resolve( rows );
+          } );
+      } );
+  }
+  close() {
+      return new Promise( ( resolve, reject ) => {
+          this.connection.end( err => {
+              if ( err )
+                  return reject( err );
+              resolve();
+          } );
+      } );
+  }
+}
+
+// Need a dotenv file with DB_HOST, DB_USER, DB_PASS, and DB_NAME
```

Comment:
Strongly suggest to create a new variable for the localhost information and change to 
- const db = new Database( process.env.JAWSDB_URL || "new_variable" )